csharp-advanced-topics/MemoryInCSharp/App/MemoryVsArrayBenchmarks.cs

```diff
@@ -0,0 +1,55 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
+namespace App;
+
+[MemoryDiagnoser]
+[RankColumn]
+[Orderer(SummaryOrderPolicy.FastestToSlowest)]
+public class MemoryVsArrayBenchmarks
+{
+    private const int Size = 10000000; // 10 million
```

Comment:
Remove comment and use _ to group digits

Reply:
@jshergal Done